examples/tenant-encryption.yaml

```diff
@@ -172,7 +172,9 @@ spec:
   #    value: storage
 
   ## Add environment variables to be set in MinIO container (https://github.com/minio/minio/tree/master/docs/config)
-  # env:
+  env:
+    - name: MINIO_KMS_AUTO_ENCRYPTION
+      value: "on" # to turn-on encryption
```

Comment:
This should be commented out by default, we shouldn't choose ENVs for users in anyway - it is upto the operator admin deploying this.

Reply:
added the configuration commented